platform/broadcom/sonic-platform-modules-ragile/common/script/device_i2c.py

```diff
@@ -253,9 +247,6 @@ def otherinit():
         if not ret:
             click.echo("%%DEVICE_I2C-INIT: init param %s failed." % index.get("name"))
 
-    for index in GLOBALINITCOMMAND:
-        log_os_system(index)
```

Comment:
>log_os_system [](http://example.com/codeflow?start=8&length=13)

why remove this part? behavior change?

Reply:
Looks like `GLOBALINITCOMMAND` is an empty list so I removed it.
Ref: All places I found that are associated with `GLOBALINITCOMMAND`
1. https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-ragile/common/script/ragilecommon.py#L334
2. https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-ragile/ra-b6510-32c/config/x86_64_ragile_ra_b6510_32c_r0_config.py#L310
3. https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-ragile/ra-b6920-4s/config/x86_64_ragile_ra_b6920_4s_r0_config.py#L228

Reply:
We may keep behavior, but enforce caller to provide `GLOBALINITCOMMAND` with each element a string array.